Makefile

```diff
@@ -85,6 +85,7 @@ NPM_REGISTRY=https://registry.npmjs.org
 # enable/disable chart repo supporting
 CHARTFLAG=false
 BUILDTARGET=build
+GEN_TLS=
```

Comment:
If in prepare it's called `gencert` why in Makefile it's called `gentls`?

Reply:
The whole name should be `generate_internal_tls_certificates`. So I think both are correct